src/translation/german.c

```diff
@@ -32,63 +32,64 @@ static translation_string all_strings[] = {
     {TR_CONFIG_LANGUAGE_DEFAULT, "(Standard)"},
     {TR_CONFIG_DISPLAY_SCALE, "Skalierung der Anzeige:"},
     {TR_CONFIG_CURSOR_SCALE, "Skalierung des Mauszeigers:"},
-    {TR_CONFIG_HEADER_UI_CHANGES, "Benutzeroberfläche ändern"},
-    {TR_CONFIG_HEADER_GAMEPLAY_CHANGES, "Spielablauf ändern"},
+    {TR_CONFIG_HEADER_UI_CHANGES, "Benutzeroberfläche"},
+    {TR_CONFIG_HEADER_GAMEPLAY_CHANGES, "Spielablauf"},
     {TR_CONFIG_SHOW_INTRO_VIDEO, "Intro-Videos abspielen"},
-    {TR_CONFIG_SIDEBAR_INFO, "Aktiviere zusätzliche Informationen im rechtsseitigen Bedienfeld"},
-    {TR_CONFIG_SMOOTH_SCROLLING, "Aktiviere sanftes Scrollen"},
+    {TR_CONFIG_SIDEBAR_INFO, "Zusätzliche Informationen am rechten Bildschirmrand anzeigen"},
+    {TR_CONFIG_SMOOTH_SCROLLING, "Sanftes Scrollen aktivieren"},
+    {TR_CONFIG_DISABLE_MOUSE_EDGE_SCROLLING, "Verschieben der Karte am Fensterrand deaktivieren"},
     {TR_CONFIG_DISABLE_RIGHT_CLICK_MAP_DRAG, "Verschieben der Karte mit der rechten Maustaste deaktivieren"},
-    {TR_CONFIG_VISUAL_FEEDBACK_ON_DELETE, "Beim Planieren der Landfläche das visuelle Feedback verbessern"},
+    {TR_CONFIG_VISUAL_FEEDBACK_ON_DELETE, "Das visuelle Feedback beim Planieren verbessern"},
     {TR_CONFIG_ALLOW_CYCLING_TEMPLES, "Erlaube alle Tempel nacheinander zu bauen"},
     {TR_CONFIG_SHOW_WATER_STRUCTURE_RANGE, "Zeige Reichweite beim Bauen von Reservoirs und Brunnen"},
     {TR_CONFIG_SHOW_CONSTRUCTION_SIZE, "Zeige Konstruktionsgröße beim Bauen"},
     {TR_CONFIG_HIGHLIGHT_LEGIONS, "Hebe Legion beim Berühren mit der Maus hervor"},
-    {TR_CONFIG_SHOW_MILITARY_SIDEBAR, "Aktiviere rechtsseitiges Bedienfeld für Legionen"},
+    {TR_CONFIG_SHOW_MILITARY_SIDEBAR, "Zeige Legionen am rechten Bildschirmrand"},
     {TR_CONFIG_FIX_IMMIGRATION_BUG, "Behebe Immigrationsfehler auf 'Sehr schwierig'"},
     {TR_CONFIG_FIX_100_YEAR_GHOSTS, "Behebe '100 Jahre alte Geister'"},
     {TR_HOTKEY_TITLE, "Julius Tastenkombinationen einstellen"},
     {TR_HOTKEY_LABEL, "Tastenkombination"},
-    {TR_HOTKEY_ALTERNATIVE_LABEL, "Alternative"},
+    {TR_HOTKEY_ALTERNATIVE_LABEL, "Alternativ"},
     {TR_HOTKEY_HEADER_ARROWS, "Pfeiltasten"},
     {TR_HOTKEY_HEADER_GLOBAL, "Globale Tastenkombinationen"},
     {TR_HOTKEY_HEADER_CITY, "Stadt"},
     {TR_HOTKEY_HEADER_ADVISORS, "Berater"},
     {TR_HOTKEY_HEADER_OVERLAYS, "Spezialkarten"},
     {TR_HOTKEY_HEADER_BOOKMARKS, "Lesezeichen für Stadtkarten"},
     {TR_HOTKEY_HEADER_EDITOR, "Editor"},
-    {TR_HOTKEY_HEADER_BUILD, "Bauen"},
+    {TR_HOTKEY_HEADER_BUILD, "Tastenkombinationen zum Bauen"},
```

Comment:
Did you check if this string fits?

Reply:
I didn't check this Version, but the one I did in Augustus previously had still space left.